bin/yamllintrules

```diff
@@ -0,0 +1,36 @@
+---
+ignore: |
+  /tests/e2e/tests/pilot/testdata/networking/v1alpha3/rule-503test-destinationrule-c.yaml
+  /tests/e2e/tests/pilot/testdata/networking/v1alpha3/destination-rule-c.yaml
+  /tests/e2e/tests/pilot/testdata/networking/v1alpha3/rule-503test-destinationrule-c-add-subset.yaml
+  /tests/e2e/tests/pilot/testdata/networking/v1alpha3/rule-503test-destinationrule-c-del-subset.yaml
+  /tests/e2e/tests/pilot/testdata/networking/v1alpha3/destination-rule-c-private.yaml
+  /galley/tools/gen-meta/metadata.yaml
+  /galley/metadata.yaml
+
+rules:
+  braces: disable
```

Comment:
why is pretty much every rule disabled?

Reply:
Will change these files under related rules.

Reply:
> Will change these files under related rules.

The above files are actually template files so they all will fail the syntax check, so we need to ignore them all. for the metadata.yaml I have put it into the related rules.